src/models/Expense.model.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
 const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.FLOAT,
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
+    },
+    note: {
+      type: DataTypes.STRING,
```

Comment:
Consider adding a length constraint to the `note` field if there is a maximum length requirement. This can prevent excessively long notes from being stored.